src/custom/state/gnosisSafe/actions.ts

```diff
@@ -0,0 +1,4 @@
+import { createAction } from '@reduxjs/toolkit'
+import { SafeInfoResponse } from '@gnosis.pm/safe-service-client'
+
+export const setSafeInfo = createAction<SafeInfoResponse | undefined>('gnosisSafe/safeInfo')
```

Comment:
May I suggest to use [jotai](https://jotai.org/) instead?

I know, mixing state management is a terrible idea.
But it is already mixed :)

It was introduced awhile ago by Uniswap in the Widgets subapp and we have been experimenting to use that in the rest of the app.
It might come a point where we switch over - that is, assuming the experiment is successful and we are happy with the results.

What do you say @anxolin, good place for using it here, no?

Reply:
Awesome! It's exactly what I was looking for!
I will add it